resources/definitions/fdmprinter.def.json

```diff
@@ -1258,6 +1258,23 @@
                     "limit_to_extruder": "wall_0_extruder_nr",
                     "settable_per_mesh": true
                 },
+                "z_seam_corner":
+                {
+                    "label": "Z Seam Corner Preference",
```

Comment:
Please remove `Z` from `Z Seam`. The seam algorithm is used for both Z-seam and retraction-seams..
The previous settings `Z Seam X`, `Z Seam Alignment` etc are kind of a misnomer.